recipes/drm4g/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "drm4g" %}
+{% set version = "3.0.11" %}
+{% set sha256 = "ece23c282d1d8fb82b34a8885166676984fd846da7ff644edc5fc174364de2a0" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/drm4g-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k or win or osx]
+  entry_points:
+    - drm4g=drm4g.commands.main:main
+    - gw_im_mad_drm4g.py=drm4g.core.im_mad:main
+    - gw_em_mad_drm4g.py=drm4g.core.em_mad:main
+    - gw_tm_mad_drm4g.py=drm4g.core.tm_mad:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: {{ build_number }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - pip
+    - python
+  run:
+    - fabric
+    - paramiko >=2.4
+    - python
+    - scp
+
+test:
+  imports:
+    - drm4g
+  commands:
+    - pip check
+    - drm4g --help
+    - gw_im_mad_drm4g.py --help
+    - gw_em_mad_drm4g.py --help
+    - gw_tm_mad_drm4g.py --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/SantanderMetGroup/DRM4G
+  summary: Meta-scheduling framework for distributed computing infrastructures
+  dev_url: https://github.com/SantanderMetGroup/DRM4G
+  license: EUPL-1.1
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - cofinoa
+    - fernanqv
```

Comment:
We need confirmation from this user that they are willing to maintain this package. Please only tick this in the checklist once it is actually done.

Reply:
To avoid more overhead the other maintainer will be removed